README.md

```diff
@@ -25,6 +25,12 @@
       ]
```

Comment:
can you add a note to the readme/amend the list of tested CLI versions above to include any other versions ? thanks!